va/caa.go

```diff
@@ -255,17 +256,17 @@ func (va *ValidationAuthorityImpl) performRemoteCAACheck(
 					result.Problem = probs.ServerInternal("Remote VA IsCAAValid RPC cancelled")
 				} else {
 					// Handle validation error.
-					va.log.Errf("Remote VA %q.IsCAAValid failed: %s", rva.Address, err)
+					va.log.Errf("Remote VA %q.IsCAAValid failed: %s", perspective, err)
```

Comment:
rva.Perspective is probably less useful than rva.Address when trying to find a remote VA instance that's acting up so you can terminate it.